### PR TITLE
Creating a Data Model

### PR DESCRIPTION
**Pull request by momo-87**
### Added Features:

- [x] Set up linters
- [x] Create a new app
- [x] Set Postgre as database adapter
- [x] Create and run the necessary migration files

### CODE REVIEW

File: db/migrate/20230815162817_create_likes.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Migration to create the Like table.
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: { to_table: :users }
```
Comment (Davidosky007) [inline db/migrate/20230815162817_create_likes.rb]: * As stated in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md#:~:text=Be%20sure%20to%20reference%20the%20ERD%20diagram%20from%20the%20sneak%20peek%20if%20you%20have%20any%20questions%20about%20table%20or%20column%20names%20or%20data%20relationships.): Be sure to reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships.   as currently, the like table have a different column name as shown in the ERD below 👇 
image

File: db/migrate/20230815163236_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Migration to create the Comment table.
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: { to_table: :users }
```
Comment (Davidosky007) [inline db/migrate/20230815163236_create_comments.rb]: * As stated in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md#:~:text=Be%20sure%20to%20reference%20the%20ERD%20diagram%20from%20the%20sneak%20peek%20if%20you%20have%20any%20questions%20about%20table%20or%20column%20names%20or%20data%20relationships.): Be sure to reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships.   as currently, the comments table have a different column name as shown in the ERD below 👇 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @momo-87 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Database migration files used 👍 
* Well written PR 👍 
* Good looking readme 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @momo-87 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**